backend/tournesol/serializers/inconsistencies.py

```diff
@@ -2,6 +2,49 @@
 from rest_framework.serializers import Serializer
 
 
+class CycleSerializer(Serializer):
+    """
+    Serializer for one cycle of length 3
+    """
+    criterion = serializers.CharField()
```

Comment:
For consistency with existing API responses, I would prefer to use `criteria` here.   
(We are not very consistent across the codebase :grimacing:, but in all serializers at least `criteria` seems to be used as an invariable noun).

Reply:
I don't think this is the right spelling in theory, but ok, it is simpler to write it as an invariable noun everywhere